x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/140_metadata.yml

```diff
@@ -0,0 +1,103 @@
+setup:
+  - requires:
+      test_runner_features: [capabilities]
+      capabilities:
+        - method: POST
+          path: /_query
+          parameters: [method, path, parameters, capabilities]
+          capabilities: [metadata_field_ignored]
+      reason: "Ignored metadata field capability required"
+
+  - do:
+      indices.create:
+        index: test
+        body:
+          settings:
+            number_of_shards: 1
+            number_of_replicas: 0
+          mappings:
+            properties:
+              integer:
+                type: integer
+                ignore_malformed: true
+              keyword:
+                type: keyword
+                ignore_above: 3
+              case:
+                type: keyword
+
+
+  - do:
+      bulk:
+        index: test
+        refresh: true
+        body:
+          - { index: { } }
+          - { case: "ok", integer: 10, keyword: "ok" }
```

Comment:
FYI: Used a "case" field to reuse the index and avoid sorting flakyness/errors

Reply:
These are the names of the test case documents? Makes sense.